pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -323,7 +329,7 @@ func (r *hibernationReconciler) Reconcile(ctx context.Context, request reconcile
 				return r.startMachines(cd, cdLog)
 			}
 			return reconcile.Result{}, nil
-		case hivev1.ResumingHibernationReason:
+		case hivev1.ResumingHibernationReason, hivev1.SyncSetsAppliedReason:
```

Comment:
I know why you had to add it here, but for a human reading english, it makes more sense to say if we should be running (aka !shouldhibernate) and hibernating condition = false but status != running, check cluster resumed... You don't have to fix it, I would take care of it in the refactor

Reply:
Yeah, I think the refactor should rename some things. Like `checkClusterResumed` should be `checkClusterRunning` -- in this case, we never hibernated, so we never resumed; and the function's job has nothing to do with resuming.

Et cetera.